### PR TITLE
[CLOB-947] Split TEST-USD liquidity tier with differing base_position_notional

### PR DESCRIPTION
**Pull request by jonfung-dydx**

### CODE REVIEW

File: protocol/testing/genesis.sh [hunk 1]
```diff
@@ -1518,11 +1518,21 @@ function update_genesis_use_test_volatile_market() {
...
-	# Liquidity Tier: For TEST-USD
+	# Liquidity Tier: For TEST-USD. 1% leverage and regular 1m nonlinear margin thresholds.
 	NUM_LIQUIDITY_TIERS=$(jq -c '.app_state.perpetuals.liquidity_tiers | length' < ${GENESIS})
 	dasel put -t json -f "$GENESIS" '.app_state.perpetuals.liquidity_tiers.[]' -v "{}"
 	dasel put -t int -f "$GENESIS" '.app_state.perpetuals.liquidity_tiers.last().id' -v "${NUM_LIQUIDITY_TIERS}"
-	dasel put -t string -f "$GENESIS" '.app_state.perpetuals.liquidity_tiers.last().name' -v 'test-usd-100x-liq-tier'
+	dasel put -t string -f "$GENESIS" '.app_state.perpetuals.liquidity_tiers.last().name' -v 'test-usd-100x-liq-tier-linear'
+	dasel put -t int -f "$GENESIS" '.app_state.perpetuals.liquidity_tiers.last().initial_margin_ppm' -v '10007' # 1% + a little prime (100x leverage)
+	dasel put -t int -f "$GENESIS" '.app_state.perpetuals.liquidity_tiers.last().maintenance_fraction_ppm' -v '500009' # 50% of IM + a little prime
+	dasel put -t int -f "$GENESIS" '.app_state.perpetuals.liquidity_tiers.last().base_position_notional' -v '1000000000007' # 1_000_000 USDC + a little prime
```
Comment (BrendanChou) [inline protocol/testing/genesis.sh:1528]: not a prime number
try `1000000000011` instead ([link](https://www.wolframalpha.com/input?i=smallest+prime+greater+than+1000000000001))

File: protocol/testing/genesis.sh [hunk 2]
```diff
@@ -1518,11 +1518,21 @@ function update_genesis_use_test_volatile_market() {
...
+	dasel put -t int -f "$GENESIS" '.app_state.perpetuals.liquidity_tiers.last().impact_notional' -v '50000000000' # 50_000 USDC (500 USDC / 1%)
+
+	# Liquidity Tier: For TEST-USD. 1% leverage and 100 nonlinear margin thresholds.
+	NUM_LIQUIDITY_TIERS_2=$(jq -c '.app_state.perpetuals.liquidity_tiers | length' < ${GENESIS})
```
Comment (BrendanChou) [inline protocol/testing/genesis.sh:1532]: Nit: I would just say `NUM_LIQUIDITY_TIERS+1` below instead of defining a new variable

Comment (jonfung-dydx) [reply protocol/testing/genesis.sh:1532]: Would have to use `NUM_LIQUIDITY_TIERS-1` in reference below 
```
	dasel put -t int -f "$GENESIS" '.app_state.perpetuals.perpetuals.last().params.liquidity_tier' -v "${NUM_LIQUIDITY_TIERS}"
```

File: protocol/testing/genesis.sh (hunk 1 above)
Comment (jonfung-dydx) [reply protocol/testing/genesis.sh:1528]: 1000000000011 is divisible by 3

Comment (jonfung-dydx) [reply protocol/testing/genesis.sh:1528]: `1000000000039` looks good

Comment (BrendanChou) [reply protocol/testing/genesis.sh]: Right, I should have gone to [this](https://www.wolframalpha.com/input?i=smallest+prime+greater+than+1000000000001_10) instead (original one interpreted my input as binary)

File: protocol/testing/genesis.sh (hunk 2 above)
Comment (BrendanChou) [reply protocol/testing/genesis.sh]: I think it was better before, might have misunderstood the comment

**Review by BrendanChou — APPROVED** [review]

File: protocol/testing/genesis.sh [hunk 3]
```diff
@@ -1518,11 +1518,21 @@ function update_genesis_use_test_volatile_market() {
...
 	dasel put -t json -f "$GENESIS" '.app_state.perpetuals.liquidity_tiers.[]' -v "{}"
 	dasel put -t int -f "$GENESIS" '.app_state.perpetuals.liquidity_tiers.last().id' -v "${NUM_LIQUIDITY_TIERS}"
-	dasel put -t string -f "$GENESIS" '.app_state.perpetuals.liquidity_tiers.last().name' -v 'test-usd-100x-liq-tier'
+	dasel put -t string -f "$GENESIS" '.app_state.perpetuals.liquidity_tiers.last().name' -v 'test-usd-100x-liq-tier-linear'
+	dasel put -t int -f "$GENESIS" '.app_state.perpetuals.liquidity_tiers.last().initial_margin_ppm' -v '10007' # 1% + a little prime (100x leverage)
+	dasel put -t int -f "$GENESIS" '.app_state.perpetuals.liquidity_tiers.last().maintenance_fraction_ppm' -v '500009' # 50% of IM + a little prime
+	dasel put -t int -f "$GENESIS" '.app_state.perpetuals.liquidity_tiers.last().base_position_notional' -v '1000000000039' # 1_000_000 USDC + a little prime
+	dasel put -t int -f "$GENESIS" '.app_state.perpetuals.liquidity_tiers.last().impact_notional' -v '50000000000' # 50_000 USDC (500 USDC / 1%)
+
+	# Liquidity Tier: For TEST-USD. 1% leverage and 100 nonlinear margin thresholds.
```
Comment (lucas-dydx) [inline protocol/testing/genesis.sh:1531]: Did you mean this?

```suggestion
		# Liquidity Tier: For TEST-USD-2. 1% leverage and 100 nonlinear margin thresholds.
```

**Review by lucas-dydx — APPROVED** [review]

**Pull request merged**